pkg/clusterresourceoverride/admission.go

```diff
@@ -109,34 +109,42 @@ func NewAdmission(kubeClientConfig *restclient.Config, stopCh <-chan struct{}, c
 	}
 
 	factory := informers.NewSharedInformerFactory(client, defaultResyncPeriod)
-	informer := factory.Core().V1().Namespaces()
-	nsLister := informer.Lister()
 
-	nsInformer := informer.Informer()
+	namespaces := factory.Core().V1().Namespaces()
+	nsInformer := namespaces.Informer()
 	go nsInformer.Run(stopCh)
 
+	limitRanges := factory.Core().V1().LimitRanges()
+	limitRangeInformer := limitRanges.Informer()
+	go limitRangeInformer.Run(stopCh)
+
 	if !cache.WaitForCacheSync(stopCh, nsInformer.HasSynced) {
-		err = fmt.Errorf("name=%s failed to wait for cache to sync", Name)
+		err = fmt.Errorf("name=%s failed to wait for Namespace informer cache to sync", Name)
 
 		klog.V(1).Info(err.Error())
```

Comment:
logging and returning an error at the same time is suspicious

Reply:
Yes, I fixed it. I don't do it for the other error handling cases above. It was an oversight.